cockatrice/src/localserver.cpp

```diff
@@ -11,6 +11,10 @@ LocalServer::LocalServer(QObject *parent)
 
 LocalServer::~LocalServer()
 {
+    // no need to lock clients, they live in the same thread
```

Comment:
`LocalServer` is just a mockup built around the abstract `Server` class, so it implements almost nothing. All the client handling is in `Server` too, except this part.
I thought that in the future someone reading this code would ask himself why clients are directly being destroyed here without proper locking, instead of in the async+lock way they are destroyed in servatrice; so i left this comment.